### ISSUE TITLE
Suggestions - nnUnet variants. Help!!

### ISSUE THREAD

**Issue by imruljubair** [MIC-DKFZ/nnUNet#1908]
Hi Good People,

I need a general suggestion for my project. The dataset I'm working on at this moment has high anatomical variability (you can see it as: the training set is very random in nature, test cases can be completely different from training distribution). I applied nnUnet, it's giving moderate results (quite expected because of the nature of the dataset). But I'd like to experiment more.

I know there are many variants of nnUnet. Can anyone suggest me any of those variants that may work better, given my dataset's nature?

Your comments will be highly appreciated.

Regards,
-
Jubair

**Comment by imruljubair** [comment]
Hi,
Still looking forward to any kind of suggestions. Thank you :)

**Comment by Kobalt93** [comment]
Hi Jubair, sorry for the late response.

If your training and test sets have a strong domain shift, I would start by increasing data augmentation, since it effectively broadens the training distribution. This can sometimes slightly reduce in-domain performance, but often improves generalization to out-of-distribution cases. You can try a stronger augmentation pipeline, such as:
https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunetv2/training/nnUNetTrainer/variants/data_augmentation/nnUNetTrainerDA5.py

Also, pay attention to normalization:
If you are using global normalization but your scanners have significantly different imaging properties (leading to inconsistent physical units), switching to per-case normalization can be beneficial, as it reduces dependency on absolute intensity values.

Hope this helps!
